vertical-pod-autoscaler/pkg/utils/metrics/updater/updater.go

```diff
@@ -18,21 +18,65 @@ limitations under the License.
 package updater
 
 import (
+	"strconv"
+
 	"github.com/prometheus/client_golang/prometheus"
 	"k8s.io/autoscaler/vertical-pod-autoscaler/pkg/utils/metrics"
 )
 
 const (
 	metricsNamespace = metrics.TopMetricsNamespace + "updater"
+
+	maxVpaSizeLog = 20
+	// maxVpaSize = 2 ^ maxVpaSizeLog
+	maxVpaSize = 1024 * 1024
```

Comment:
That seems to be a looot for one deployment. Which is a good thing I guess

Reply:
Yes, it's more than max number of Pods in a cluster (5K nodes * 110 pods/node) so I hope we don't have to change it anytime soon.